test/smoke/testApps/PerfTestAppSpringBoot/src/main/java/com/microsoft/applicationinsights/testapps/perf/boot/controllers/FakeRestController.java

```diff
@@ -38,11 +38,9 @@ public void run() {
                         ? pUrl
                         : "http://"+pUrl);
 
-                CloseableHttpClient client = HttpClients.createDefault();
-                try {
+                try (CloseableHttpClient client = HttpClientBuilder.create().disableAutomaticRetries().build()) {

```

Comment:
This is much clearer :) We should make this refactoring in core sdk code base also.